docs/propertyestimator.rst

```diff
@@ -45,7 +45,7 @@ This API can be extended in the future to provide access to the simulation data
 .. code-block:: python
 
     # Attach to my compute and storage resources
-    estimator = PropertyEstimator(...)
+    estimator = PropertyEstimatorClient(...)
```

Comment:
Does it make sense to call it `PropertyEstimatorClient()` if this is the end that the user sees? I definitely understand the idea of having a `PropertyEstimatorServer` class on the backend, but why not just call the client `PropertyEstimator` if this is what users will interact with?

We can also later move toward a usage model where, for simple calculations, the user doesn't even need to spin up a `PropertyEstimatorServer()` manually---the software can spin up a server on the current workstation if none is running or specified to connect to.

Reply:
I personally prefer it with Client because it makes it more obvious that this isn't just built around one class on one machine, but rather this will simply be the class used to interact with the main compute system. I believe this is the naming convention @dgasmith went with for QCFractal. Id be happy to drop the name though if people don't agree!

Reply:
`DaskClient`, `MongoClient`, etc common and explicit.